src/pages/add/import-wallet/import-wallet.ts

```diff
@@ -186,12 +186,11 @@ export class ImportWalletPage {
       return undefined;
     }
     if (info.type == '1' && info.hasPassphrase) {
-      const title = this.translate.instant('Error');
+      const title = this.translate.instant('Warning');
       const subtitle = this.translate.instant(
         'Password required. Make sure to enter your password in advanced options'
       );
       this.popupProvider.ionicAlert(title, subtitle);
-      return undefined;
```

Comment:
What happen in this case if not return `undefined`? Will it return `info` anyway?

Reply:
Yes, so the words will be completed in mnemonics field.
If you are importing a wallet with mnemonics with passphrase, the import page will show you a message saying that don't forget to put the password.